tests/intfutil_test.py

```diff
@@ -281,7 +281,7 @@ def test_show_interfaces_autoneg_status_Ethernet0(self):
         assert result.exit_code == 0
         assert result.output == show_interface_auto_neg_status_Ethernet0_output
   
-    def test_show_interfaces_autoneg_status_etp9_in_alias_mode(self):
+    def test_show_interfaces_autoneg_status_eth9_in_alias_mode(self):
```

Comment:
Why do you need to change the test case name to eth9 when it is using etp9?

Reply:
I thought it's a typo, and now it's reverted